ambry-api/src/main/java/com.github.ambry/clustermap/ClusterManagerWriteStatusDelegate.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.clustermap;
+
+/**
+ * Delegate class giving a layer of abstraction between the client and ClusterParticipant
+ */
+public class ClusterManagerWriteStatusDelegate {
```

Comment:
I was thinking about this and I think this should be named `ClusterParticipantWriteStatusDelegate`.

`ClusterManager` is a component used by both frontends and datanodes to mostly get the state of the cluster.

`ClusterParticipant` is the component associated with the datanodes (what constitutes the cluster) and any changes that a datanode should make to its state should really go through the participants. In fact, that is exactly what you are doing here.

Reply:
Yeah sure, changed